code/modules/zombie/organs.dm

```diff
@@ -0,0 +1,91 @@
+#define START_TIMER reanimation_timer = world.time + rand(600,1200)
+
+/obj/item/organ/internal/zombie_infection
+	name = "festering ooze"
+	desc = "A black web of pus and viscera."
+	parent_organ = "head"
+	slot = "zombie_infection"
+	icon_state = "blacktumor"
+	var/causes_damage = TRUE
+	var/reanimation_timer
+	var/datum/species/old_species = new /datum/species/human
+	var/living_transformation_time = 5
```

Comment:
Why 5 instead of 30 living transformation time? Is this testing variable left in?

Reply:
Should be 3 actually, TG uses deciseconds for their stuns but we use ticks, if we called stun(30) on a zombie they wouldn't get up anytime soon